src/main/java/de/metas/ui/web/window/model/sql/SqlDocumentQueryBuilder.java

```diff
@@ -414,8 +415,12 @@ private String getSqlWhere(final List<Object> sqlParams)
 			// Composed primary key
 			else
 			{
-				sqlWhereClauseBuilder.appendIfNotEmpty("\n /* key */ AND ");
-				sqlWhereClauseBuilder.append("(");
+				final boolean parenthesesRequired = !sqlWhereClauseBuilder.isEmpty();
+
+				if (parenthesesRequired)
```

Comment:
brainfart of the day; next time *i* come across the need to put something into branckes, i think i will use our `IAutoCloseable`